src/_stylesheets/_variables.scss

```diff
@@ -1,2 +1,9 @@
 $app-light-blue: govuk-tint($govuk-brand-colour, 95%);
 $app-blue-shade-25: govuk-shade($govuk-brand-colour, 25%);
+
+$app-green: #11875a;
+
+$app-black: govuk-colour('black');
```

Comment:
I was more thinking just using `govuk-colour('black')` directly in the scss rather than via an `app` variable. Same with the tints below, although there's probably a case for that given we're using tints in these variables for other colours. I don't believe this specific variable is being used anywhere though so I'd make the case to just scrap it.

Reply:
I dunno, it feels neater to me to only make the `govuk-colour` call out to Frontend once, and then use the locally assigned variable for the following uses. Maybe it's just 'performance theatre' that makes little actual difference.